src/Raven.Client/Documents/Session/IAbstractDocumentQuery.cs

```diff
@@ -19,6 +19,8 @@ public interface IAbstractDocumentQuery<T>
     {
         string IndexName { get; }
 
+        bool IsFilterActive { get; }
```

Comment:
Do we really need to make it as part of `IAbstractDocumentQuery` interface? That will make it available to users

Reply:
Yes, without it I can't build a `group by` with `count()`.